### DISCUSSION TITLE
Welcome to music_control Discussions!

### DISCUSSION THREAD

**Discussion by charlyb01** [sf-inc/music_control#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by charlyb01** [comment]
And feel free to open new Discussions to share your ideas or talk about anything related to this mod =)

**Comment by NakumaErebos** [comment]
I’ve been working with Minecraft music and texture packs for a while. I recently discovered your mod and I really love the new features it adds, especially the ability to assign soundtracks to specific conditions.

I wanted to suggest a potential improvement: it would be really powerful if it were possible to set a soundtrack to play only when multiple conditions are met, not just one. This could open up a lot of creative possibilities for people using the mod. And could you maby add the conditions village and monsters nearby?

Thank you for your work on this project
